src/fungible_token.rs

```diff
@@ -421,7 +421,20 @@ impl FungibleToken {
     }
 
     pub fn accounts_insert(&self, account_id: &str, amount: Balance) {
-        sdk::save_contract(&Self::account_to_key(account_id), &amount)
+        if self.accounts_contains_key(account_id) {
```

Comment:
Shouldn't this be if the account is _not_ known then increment the counter instead of if the account _is_ known then increment the counter?

Also, is it possible to delete an account? If so should the counter be decremented in this case?

Reply:
Fixed